### ISSUE TITLE
[REQUEST]: Please create a new release with master's latest code.

### ISSUE THREAD

**Issue by carlostrianag** [WenchaoD/FSPagerView#359]
As part of our efforts of keeping the best practices in our code, we'd like to be able to pick a specific version when using Swift Package Manager.

This is not currently possible at this time since the working code is being fetched directly from the master branch. It has been working so far, but say in a future you make a change, that change will potentially break our code just by the fact that is being fetched directly from master.

Hope you've understood the point!

In case you don't know how to make a Github Release I'd be than glad to help!

Best,

**Comment by alobaili** [comment]
I agree. A simple tag to the latest commit from `master` is sufficient to follow semantic versioning best practices. The last tag was https://github.com/WenchaoD/FSPagerView/releases/tag/0.8.3 which is very old compared to the changes made on `master`. Many SwiftPM clients rely on the "up to next major" configuration when adding this library as a dependency.
